src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -524,6 +526,24 @@ public function getConnection()
     {
         return $this->connection;
     }
+    
+    /**
+     * Execute the given callback using the given connection as the default connection.
+     *
+     * @param  string  $name
+     * @param  callable  $callback
+     * @return mixed
+     */
+    public function usingConnection($name, callable $callback)
```

Comment:
Which PR was this method originally introduced in?

Reply:
Here: https://github.com/laravel/framework/commit/f4bd05fc1015a1a49dd6850d6817a10d129143c8